test/ledger/grants.integration.test.js

```diff
@@ -0,0 +1,109 @@
+import { serial as test } from 'ava'
+import BigNumber from 'bignumber.js'
+import crypto from 'crypto'
+import anonize from 'node-anonize2-relic'
+import { v4 } from 'uuid'
+import { sign } from 'http-request-signature'
+import tweetnacl from 'tweetnacl'
+import {
+  ledgerAgent,
+  uint8tohex,
+  timeout,
+  ok
+} from '../utils'
+const promotionId = 'c96c39c8-77dd-4b2d-a8df-2ecf824bc9e9'
+test('ledger: grants: add expired grant', async t => {
```

Comment:
I would bundle this and the below test together. generally for the non-e2e test have been trying to keep the tests self contained and not dependent on other tests to run